### ISSUE TITLE
cmake build fails with wxUSE_GTKPRINT=ON

### ISSUE THREAD

**Issue by antonio-rojas** [wxWidgets/wxWidgets#22745]
**Describe the problem**
Building with cmake and wxUSE_GTKPRINT=ON fails, because the gtkunixprint.h header location is not added to the include dirs
```
/build/wxwidgets/src/wxWidgets-3.2.0/src/gtk/print.cpp:37:10: fatal error: gtk/gtkunixprint.h: No such file or directory
   37 | #include <gtk/gtkunixprint.h>
      |          ^~~~~~~~~~~~~~~~~~~~
compilation terminated.
```

**Platform and version information**
 - wxWidgets version you use: 3.2.0
 - wxWidgets port you use: wxGTK
 - OS and its version: Linux x86_64
 - Compiler being used: GCC 12.2

**Comment by vadz** [comment]
Thanks for reporting this.

We probably just need `pkg_check_modules(gtk+-unix-print-${TOOLKIT_VERSION}.0)` somewhere, but I'm not sure where exactly. Any PRs would be welcome.

**Comment by MaartenBent** [comment]
Indeed. CMake doesn't have any of the `gtkprint` checks that configure has. This part of configure has to be converted to CMake:
https://github.com/wxWidgets/wxWidgets/blob/6ee593c78d082156c4fba2997241e530ba1d24da/configure.in#L5834-L5856

I can create a simple `FindGTKPRINT.cmake` module that has a `pkg_check_modules`, and use that to find the headers and libraries.

**Comment by vadz** [comment]
Fixed in both master and 3.2, thanks again, Maarten!
